web-ui/src/pages/PulseReportPage.jsx

```diff
@@ -376,25 +514,25 @@ const PulseReportPage = () => {
             selected={teamMembers}
           />
           {lineChart()}
+          {averageScores()}
           {barChart()}
+          <Modal open={showComments} onClose={() => setShowComments(false)}>
```

Comment:
This dialog displays the internal and external comments associated with a selected pulse entry (by click its comment icon).